src/test/standardTest.node.ts

```diff
@@ -58,7 +58,7 @@ async function installPythonExtension(vscodeExecutablePath: string) {
     // Pick python extension to use based on environment variable. Insiders can be flakey so
     // have the capability to turn it off/on.
     const pythonVSIX =
-        process.env.VSC_JUPTYER_PYTHON_EXTENSION_VERSION === 'insiders'
+        process.env.VSC_JUPYTER_PYTHON_EXTENSION_VERSION === 'insiders'
```

Comment:
I'm not sure why this is labeled as a change 😅 I can't see the difference 😵

Reply:
JUPYTER vs JUPTYER , the `TY` are swapped.
I'm 200% certain I introduced this typo